src/js/index.js

```diff
@@ -1,21 +1,39 @@
 import '../less/style.less';
 import Maze from './Maze.js';
 
-const mazeRows = 10;
-const mazeCols = 10;
+const nRows = 4;
+const nCols = 6;
 
 $(document).ready(() => {
-    setupView();
 
-    const maze = new Maze(mazeRows, mazeCols);
+
+    const maze = new Maze(nRows, nCols);
     const cells = maze.getCells();
-    const lastCell = cells[cells.length-1];
-    console.log(lastCell.getWalls());
+    const lastCell = cells[cells.length-3];
+    // console.log(lastCell.getWalls());
+    const neighs = lastCell.getAccessibleNeighbors();
+    console.log(neighs)
+
+    // const cell1 = cells[7]
+    // const cell2 = cells[1]
+    // const cs = [cell1, cell2]
+
+
+    setupView(maze);
+    Object.values(neighs).forEach((c)=>{
+      if(c!==null){
+        $(`#${c.row+'-'+c.col}`).css("background-color","blue")
+      }
+
+    })
```

Comment:
just leaving in all this console logging stuff 'cause it helps with testing at this point